r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -711,3 +711,34 @@ test_that("am/pm mirror lubridate", {
   )
 
 })
+
+test_that("extract tz", {
+  df <- tibble(
+    posixct_date = as.POSIXct(c("2022-02-07", "2022-02-10"), tz = "Pacific/Marquesas"),
```

Comment:
I cleaned up the types that we aren't running through the dplyr pipeline here — we can add them in if/when we want to test that mechanism.